pkgs/development/ocaml-modules/ppx_deriving/default.nix

```diff
@@ -10,31 +10,25 @@
 , ounit
 , ounit2
 , ocaml-migrate-parsetree
-, ocaml-migrate-parsetree-2
 }:
 
 let params =
   if lib.versionAtLeast ppxlib.version "0.20" then {
     version = "5.2.1";
     sha256 = "11h75dsbv3rs03pl67hdd3lbim7wjzh257ij9c75fcknbfr5ysz9";
-    useOMP2 = true;
   } else if lib.versionAtLeast ppxlib.version "0.15" then {
     version = "5.1";
     sha256 = "1i64fd7qrfzbam5hfbl01r0sx4iihsahcwqj13smmrjlnwi3nkxh";
```

Comment:
Does 5.1 not need omp2?

Reply:
It needs OMP, but at version 1.

`useOMP2` is false for version 5.1 and that line is rather explicit: https://github.com/ocaml-ppx/ppx_deriving/blob/v5.1/ppx_deriving.opam#L21